rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_17.json

```diff
@@ -66,30 +66,30 @@
                                                     "supply_fans": [
                                                         {
                                                             "id": "Supply Fan 1",
-                                                            "design_airflow": 534.7164531455998,
                                                             "specification_method": "SIMPLE",
-                                                            "design_electric_power": 35
+                                                            "design_electric_power": 60,
+                                                            "design_airflow": 471.94744319999984
```

Comment:
To pass rule 17-17-a, the sum of fan powers /  supply airflow <= 0.3 (W/cfm). 

Could you slightly increase the ```design_airflow``` value? This is because currently, what you did is correct, however because of the conversion factor, the ```fan_power_W_CFM``` becomes 0.3000000000000002 watt / cfm, which becomes false.

Reply:
While this is an easy edit to make to the TCD, I think it would be better for this to be corrected in a way that does not result in failed outcomes due to insignificant rounding of digits in conversion calculations. Can we implement rounding of the results, or an allowable margin of error?

Reply:
There is a ```std_equal``` function to compare if two values are the same with margins (default 5%). However, we don't consider the margin when we compare if one value is greater/less than the other. I believe this is the first time to face this issue.

As you said, we can update our code to allow the margin or error (we also may need to dive more into the ```pint``` package to see whether there is still this issue in the newer version). However, the right procedure would be to bring up this topic during the meeting and discuss this with other team members. Could you do this during the TCD meeting? If accepted, we should update all the parts in the code that used ```>```, ```<```, ```>=```, and ```<=```.

While doing this, could we update the value to move forward?

Reply:
The TCD meeting was cancelled because it no longer seemed necessary after Karen and I were up to speed but I will email the team to get feedback, and I will update this value so that it passes testing. Thanks!

Reply:
Thank you Jackson!